internal/ingress/controller/controller.go

```diff
@@ -481,6 +482,7 @@ func (n *NGINXController) getBackendServers(ingresses []*extensions.Ingress) ([]
 						CorsConfig:           anns.CorsConfig,
 						ExternalAuth:         anns.ExternalAuth,
 						Proxy:                anns.Proxy,
+						ProxyPass:			  anns.ProxyPass,
```

Comment:
Is this indentation a GitHub glitch or did you forget running gofmt?

Reply:
Whoops - forgot to run gofmt, as I'm used to running it via `make` 😛